Makefile

```diff
@@ -81,7 +81,10 @@ tests-modules: ## Run unit tests for modules hooks and templates.
 
 tests-matrix: ## Test how helm templates are rendered with different input values generated from values examples.
   ##~ Options: FOCUS=module-name
+	rm -f ee/modules/030-cloud-provider-openstack/candi ee/modules/030-cloud-provider-vsphere/candi
 	go test ./testing/matrix/ -v
+	ln -s /deckhouse/candi/cloud-providers/openstack/ ee/modules/030-cloud-provider-openstack/candi
```

Comment:
TODO: Need to find a way to maintain these symlinks.

Reply:
Yes. I add comment to makefile

Reply:
as an option: after all the tests are run before the build, we can remove the symlinks, and add them at the build stage.

Reply:
I just described my ideas in the https://github.com/deckhouse/deckhouse/issues/2217